CodeEdit/AppDelegate.swift

```diff
@@ -121,6 +122,27 @@ final class AppDelegate: NSObject, NSApplicationDelegate, ObservableObject {
         }
     }
 
+    /// Handle urls with the form `codeedit://file/{filepath}:{line}:{column}`
+    func application(_ application: NSApplication, open urls: [URL]) {
+        for url in urls {
+            let file = URL(fileURLWithPath: url.path).path.split(separator: ":")
+            let filePath = URL(fileURLWithPath: String(file[0]))
+            let line = file.count > 1 ? Int(file[1]) ?? 0 : 0
+            let column = file.count > 2 ? Int(file[2]) ?? 1 : 1
```

Comment:
Why is this by default 1, a more natural approach is line 0, column 0.

Reply:
The first line in CodeEdit is labeled as `1`. This is the general standard; in Xcode the position at the beginning of the file is (1,1). GitHub is the same. So there is no line 0 or column 0.